apps/web/src/pages/integrations/CreateProviderPage.tsx

```diff
@@ -17,16 +17,20 @@ export function CreateProviderPage() {
   }
 
   return (
-    <SideBarWrapper>
-      <CreateProviderInstanceSidebar
-        goBack={() => {
+    <CreateProviderInstanceSidebar
+      onGoBack={() => {
+        if (location.pathname.includes('/integrations/create/')) {
           navigate('/integrations/create');
-        }}
-        onClose={() => {
-          navigate('/integrations');
-        }}
-        provider={foundProvider}
-      />
-    </SideBarWrapper>
+
+          return;
+        }
+
+        navigate('/integrations');
```

Comment:
Let's use the routes enum and also add to the routes the one for creating the integration:
https://github.com/novuhq/novu/blob/8e71c9b09144b5f950c6d8317e2339964051bd93/apps/web/src/constants/routes.enum.ts#L1